content/en/docs/components/pipelines/v2/installation/quickstart.md

```diff
@@ -16,30 +16,17 @@ summary {
 </style>
 
 <!-- TODO: add UI screenshots for final pipeline -->
-This tutorial helps you get started with a KFP deployment and a pipeline created with the KFP SDK.
-
-Before you begin, you need the following prerequisites:
-
-* **An existing Kubernetes cluster**: If you don't have a Kubernetes cluster, see [Installation][installation] for instructions about how to get one.
-
-* **The [kubectl](https://kubernetes.io/docs/tasks/tools/) command-line tool**: Install and configure your [kubectl context](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) to connect with your cluster.
-
-* Run the following script to install the KFP SDK:
-  ```shell
-  pip install kfp
-  ```
-
-After you complete the prerequisites, click each step to view the instructions:
```

Comment:
I think these prerequisites are still needed, because users need to have a k8s cluster before deploying KFP.